src/WebJobs.Extensions/Extensions/ExtensionsWebJobsStartup.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Extensions;
+using Microsoft.Azure.WebJobs.Hosting;
+using Microsoft.Extensions.Hosting;
+
+[assembly: WebJobsStartup(typeof(ExtensionsWebJobsStartup))]
+
+namespace Microsoft.Azure.WebJobs.Extensions
+{
+    public class ExtensionsWebJobsStartup : IWebJobsStartup
+    {
+        public void Configure(IHostBuilder builder)
+        {
+            builder.AddTimers();
+            builder.AddFiles();
```

Comment:
Do we always want to have those bundled? In the case of script, do we want files?

Reply:
These setup classes are designed to auto register all extensions for the extension assembly. While we're not using Files currently in Functions, it won't hurt anything.

Reply:
Hence my question. We'll have the Files extension registered in Functions, which will also make it available (and supported) for use there. Do we want that?